### PR TITLE
Kubernetes 1.19

### PR DESCRIPTION
**Pull request by pohly**
This is a major cleanup of how we handle deployment files, leading up to support for two flavors of deployment files for Kubernetes 1.19 (with and without alpha features).

### CODE REVIEW

**Comment by pohly** [comment]
@avalluri: please review, it passed all testing (both pre- and post-merge).

Some of the commits were already merged as part of https://github.com/intel/pmem-csi/pull/728. I'd prefer to avoid a rebase + retest, but can also do that if you want.

File: docs/install.md [hunk 1]
```diff
@@ -773,6 +795,37 @@ EOF
...
+### Storage capacity tracking
+
+[Kubernetes
+1.19](https://kubernetes.io/blog/2020/09/01/ephemeral-volumes-with-storage-capacity-tracking/)
+introduces support for publishing and using storage capacity
+information for pod scheduling. PMEM-CSI must be deployed
+differently to use this feature:
+
+- `external-provisioner` must be told to publish storage capacity
+  information via command line arguments.
```
[the remark this answers is not in the document]
Comment (pohly) [reply docs/install.md:808]: It's not required. Storage capacity tracking is an alpha feature in Kubernetes, we do not have (and probably shouldn't) support it in the operator until some time later.

[the remark this answers is not in the document]
Comment (pohly) [reply docs/install.md:808]: Tracked in https://github.com/intel/pmem-csi/issues/730

[the remark this answers is not in the document]
Comment (avalluri) [reply docs/install.md:808]: Ok.

**Pull request merged**